var/spack/repos/builtin/packages/py-coloredlogs/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
```

Comment:
Can we remove this line?

Reply:
I removed that line.

Reply:
Thanks!

Reply:
Sorry, I think you removed the line above the one I marked. I know it's a bit of an OCD, but all the other packages don't have it :sweat_smile: